### PR TITLE
Parametrize pipelinerun pruning

### PR DESCRIPTION
**Pull request by lkolacek**
Signed-off-by: Ladislav Kolacek <lkolacek@redhat.com>

### CODE REVIEW

**Review by mkosiarc — APPROVED** [review]

File: tasks/configure_namespace.yml [hunk 1]
```diff
@@ -114,8 +114,8 @@
       metadata:
         name: pipelinerun-cleaner
       spec:
-        # every 5 min
-        schedule: "*/5 * * * *"
+        # Every day at midnight
+        schedule: "0 0 * * *"
```
Comment (rcerven) [inline tasks/configure_namespace.yml:118]: this should be parametrized from group_vars

**Review by rcerven — CHANGES_REQUESTED** [review]

File: tasks/configure_namespace.yml [hunk 2]
```diff
@@ -114,8 +114,8 @@
       metadata:
         name: pipelinerun-cleaner
       spec:
-        # every 5 min
-        schedule: "*/5 * * * *"
+        # Every day at midnight
+        schedule: "{{ pipeline_run_pruning_schedule }}"
```
Comment (MartinBasti) [inline tasks/configure_namespace.yml:118]: Should role define default?
Should it be documented?

File: tasks/configure_namespace.yml [hunk 3]
```diff
@@ -114,8 +114,8 @@
       metadata:
         name: pipelinerun-cleaner
       spec:
-        # every 5 min
-        schedule: "*/5 * * * *"
+        # Every day at midnight
```
Comment (rcerven) [inline tasks/configure_namespace.yml:117]: remove that comment

Comment (MartinBasti) [inline tasks/configure_namespace.yml:117]: :eyes:

File: tasks/configure_namespace.yml (hunk 2 above)
Comment (lkolacek) [reply tasks/configure_namespace.yml:117]: We don't use default to fail horribly when groupvars are not provided.
Yes nice catch, I documented it in README.

**Review by MartinBasti — APPROVED** [review]

**Review by rcerven — APPROVED** [review]

**Pull request merged**